plugins/librato/measurement.go

```diff
@@ -38,6 +38,10 @@ func newMeasurement() *measurement {
 }
 
 func (m *measurement) floorTime(interval int64) {
+	if interval <= 0 {
+		log.Errorf("Attempted to floor measurement time with interval %d", interval)
+		return
```

Comment:
not sure if it's a good idea to just return,
maybe if interval <= 0, you can assume it's = 1? otherwise you probably should return error to the caller